pkg/envoy/rds/response.go

```diff
@@ -72,8 +63,7 @@ func NewResponse(cataloger catalog.MeshCataloger, proxy *envoy.Proxy, discoveryR
 		ingressTrafficPolicies = trafficpolicy.MergeInboundPolicies(ingressTrafficPolicies, ingressPolicy.HTTPRoutePolicies...)
```

Comment:
Notice that on a lot of these we call GetIngressTrafficPolicy (or inboundTrafficPolicy, outboundTrafficPolicy), but then we only access one field. That means we should probably add a method to just get that one field, so we don't perform the extra work.

No need to tackle in this PR but probably worth a TODO